modules/ROOT/pages/tutorial/access-control.adoc

```diff
@@ -240,62 +1147,80 @@ The complete set of privileges available to users assigned the `itadmin` role ca
 ----
 SHOW ROLE itadmin PRIVILEGES AS COMMANDS;
 ----
+
+.Result
+[%collapsible]
+====
+[queryresult]
 ----
 +-------------------------------------------------------------------------+
 | command                                                                 |
 +-------------------------------------------------------------------------+
+| "DENY CREATE ON GRAPH `healthcare` RELATIONSHIP DIAGNOSIS TO `itadmin`" |
+| "DENY CREATE ON GRAPH `neo4j` RELATIONSHIP DIAGNOSIS TO `itadmin`"      |
+| "DENY PRIVILEGE MANAGEMENT ON DBMS TO `itadmin`"                        |
+| "DENY READ {ssn} ON GRAPH `healthcare` NODE Patient TO `itadmin`"       |
+| "DENY READ {ssn} ON GRAPH `neo4j` NODE Patient TO `itadmin`"            |
```

Comment:
These 
```
"DENY CREATE ON GRAPH `neo4j` RELATIONSHIP DIAGNOSIS TO `itadmin`"
"DENY READ {ssn} ON GRAPH `neo4j` NODE Patient TO `itadmin`"
```

seems odd, as there the set of privileges assigned above specify graph `healthcare`

Reply:
Hmm, this is what the command returned. I just copied and pasted it. But you're right, this seems odd. The `itadmin` hasn't denied anything on `neo4j`. I have no idea.

Reply:
I haven't looked at the scripts, but seems to me `itadmin` probably got a 
`DENY CREATE ON GRAPH * RELATIONSHIPS DIAGNOSIS TO itadmin`  and a 
`DENY READ {ssn} ON GRAPH * NODES Patient TO itadmin` 

instead of 
`DENY CREATE ON GRAPH healthcare RELATIONSHIPS DIAGNOSIS TO itadmin` and 
`DENY READ {ssn} ON GRAPH healthcare NODES Patient TO itadmin` 
which are stated above